docs/reference/cli/players.md

```diff
@@ -0,0 +1,605 @@
+
+The `disconnect` command tells a Squeezebox 2 or newer player on another server instance to disconnect from its server and connect to us. This is the opposite of `connect`, where we tell a player connected to us to connect to a different server.
+
+Supported values are:
+
+>ip - A dotted IP address to connect to.
+
+Example:
+```
+Request: "disconnect bd:a5:a9:9b:9d:df 192.168.1.10<LF>"
+Response: "disconnect bd:a5:a9:9b:9d:df 192.168.1.10<LF>"
+```
+
+***
+## `players`
+
+`players <start> <itemsPerResponse>`
+
+The `players` query returns information about all `players` (physical players as well as streaming clients) known by the server.
+
+* `players 0` will return info about all players.
+* `players 3 2` will return information about two players starting with playerindex 3
+* `players 2 1` will return info about the player with playerindex 2
+
+
+**Accepted tagged parameters:**
+
+| Tag | Description |
+|:---|:---|
+| playerprefs | Comma separated list of preference values to return (for each player). |
+
+
+**Returned tagged parameters:**
+
+Results returned as two blocks.
+
+* First a block containing just the count of players
+* Secondly a standard set of data for each player
+    * Info for each player can in addition include values as per the list of preferences included in the optional playerprefs parameter.
+
+| Block | Tag | Description |
+|---|---|---|
+| First block |||
+|| count | Number of players known by the server. Equivalent to `player count ?`. |
+| For each player:|||
+|| playerindex | Player index. Values 0, 1, 2 etc |
+|| playerid | Player unique identifier. Equivalent to `player id <playerindex> ?`. |
+|| uuid | Player uuid. Equivalent to `player uuid <playerindex> ?`. |
+|| ip | Player IP and port. Equivalent to `player ip <playerindex|playerid> ?`. |
+|| name | Player name. Equivalent to `player name <playerindex|playerid> ?`. |
+|| seq_no | *Not clear what this is.* |
+|| model | Player model. Values like `squeezebox3`, `baby`, `squeezelite`. Equivalent to `player model <playerindex|playerid> ?`. |
+|| modelname | Model name, values like `Squeezebox Classic`, `Squeeezebox Radio`, `Squeezelite-X`. |
+|| power | Indicatess if player is powered on. Returns 1 for True, 0 for False. Equivalent to `<playerid> power ?`. |
+|| isplaying | Indicates if player currently playing. Returns 1 for True, 0 for False. |
+|| displaytype | Player display type. Not returned for streaming connections. Equivalent to `player displaytype <playerindex|playerid> ?`. |
+|| isplayer | Whether a player is a known player model. Will return 0 for streaming connections. Equivalent to `player isplayer <playerindex|playerid> ?`. |
+|| canpoweroff | Whether the player can be powered off. This value is false for streaming connections. Equivalent to `player canpoweroff <playerindex|playerid> ?` |
+|| connected | Connected state. Equivalent to `<playerid> connected ?`. |
+|| firmware | Firmware version |
+| For each defined pref requested:-|||
```

Comment:
Got an extra dash here after the semicolon

Reply:
The text " For each defined pref requested:-" is not code. It is a comment within a "Returned Tagged Parameters": 
![image](https://github.com/LMS-Community/lms-community.github.io/assets/42010857/2dbdc3de-3409-4565-91a5-93b927d1103d)

Reply:
I meant to point out the dash after the semicolon. That shouldn't be there right?

Reply:
I am going to try the fix suggested by @mherger to get the block comments to "rowspan". At same time I will check that those block comments are formatted consistentently, so example

- First block:
- For each album: